Code/GraphMol/FMCS/MaximumCommonSubgraph.cpp

```diff
@@ -960,11 +963,16 @@ MCSResult MaximumCommonSubgraph::find(const std::vector<ROMOL_SPTR>& src_mols) {
     Parameters.AtomCompareParameters.RingMatchesRingOnly = true;
   }
 
+  unsigned i = 0;
+  boost::dynamic_bitset<> faked_ring_info;
+  faked_ring_info.resize(src_mols.size());
```

Comment:
This doesn't need to be a two-step process:
```suggestion
  boost::dynamic_bitset<> faked_ring_info(src_mols.size());
```

Reply:
Done!